### PR TITLE
adding the sentimen analysis model to research folder's README.md file

### PR DESCRIPTION
**Pull request by sculd**

### CODE REVIEW

File: research/README.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ request.
 -   [rebar](rebar): low-variance, unbiased gradient estimates for discrete
     latent variable models.
 -   [resnet](resnet): deep and wide residual networks.
+-   [sentiment_analysis](sentiment_analysis): a simple model to classify
+    document's sentiment.
```
Comment (karmel) [inline research/README.md]: nit: a document's

Comment (sculd) [reply research/README.md]: ack, I will add myself to the codeowners as well.

**Comment by sculd** [comment]
ping

**Comment by jaeyounkim** [comment]
Updated the README.md

**Pull request closed without merging**